src/components/calendar/drag-drop-provider.tsx

```
@@ -0,0 +1,353 @@
+      scrollBy(dir * AUTO_SCROLL_STEP);
+    },
+    [scrollBy, viewportHeightRef]
+  );
+
+  return {
+    scrollBy,
+    onDragUpdate,
+  };
+}
+
+function useDropCompletion(
```

ESLint

[error] Function 'useDropCompletion' has too many parameters (4). Maximum allowed is 3.

(max-params)